vulnerable_people_form/templates/check-contact-details.html

```diff
@@ -6,7 +6,8 @@
 {% block form_fields %}
   <h1 class="govuk-heading-l govuk-!-margin-top-0 govuk-!-margin-bottom-3">
     {{title_text}}
-</h1>
+    </h1>
+    <p class="govuk-body">Did you mean <strong>{{ values.get("email") }}?</strong></p>
```

Comment:
remove this as it will be implemented in another ticket

Reply:
done